src/utils/helpers.ts

```diff
@@ -1,2 +1,28 @@
+// Simple Sleep function: Pauses execution for time in milliseconds
 export const sleep = (timeInMs: number) =>
   new Promise(resolve => setTimeout(resolve, timeInMs));
+
+// Rounds any number or number thats a string to a string of 2 decimal places
+export function round2Fixed(value: any) {
```

Comment:
Please annotate return values for top-level functions- especially for module exports.
```suggestion
export function round2Fixed(value: any): string | typeof NaN {
```
Also, as `NaN` can propagate in destructive ways, consider not using it unless you want those properties.